### ISSUE TITLE
Bottom toolbar is hidden behind the keyboard

### ISSUE THREAD

**Issue by AbraamNader** [kiwibrowser/src.next#222]
**Describe the bug**
Bottom toolbar is hidden behind the keyboard, until I start typing at least 2 characters, which makes entering/editing the url very hard since I have to remove the 2 characters I just entered.

**To Reproduce**
1. Enable bottom toolbar
2. Open a new tab or click on the url bar for an existing tab.
3. The keyboard pops up, and the bottom toolbar stays in place behind it.

Workaround: 
- Type 2 characters, the bottom toolbar shows up above the keyboard.

**Expected behavior**
The toolbar should be shown above the keyboard upon opening a tab or editing the url.

**Smartphone (please complete the following information):**
 - Device: OnePlus 5T
 - Android Version : Oxygen OS 10.0.1
 - Kiwi Browser Version: 93.0.4577.18
 - Gboard Version: 10.7.05

**Additional context**
- I use the Oxygen OS bottom navigation fullscreen gestures, so the navigation/notification bars are hidden.
- The bottom toolbar is the main reason I use Kiwi, so much easier to reach, also maximizes the visible content area, thanks for keeping it.

**Comment by karl-pae** [comment]
Can you share the screenshot. I don't have such issue.

**Comment by AbraamNader** [comment]
> Can you share the screenshot. I don't have that issue.

Here you go
Hidden behind the keyboard:

![hidden](https://user-images.githubusercontent.com/899978/127530256-1107817a-7d81-45c9-961f-bbdbbef8df1e.jpg)



After typing 2 characters:

![shown](https://user-images.githubusercontent.com/899978/127530380-d3956db2-69de-45da-a997-5992fde473b9.jpg)

**Comment by karl-pae** [comment]
Does it still occur with recent release Generation 1087859047 ?

**Comment by AbraamNader** [comment]
Just tried release Generation 1087859047 and it has the same issue, 
however I figured it out, as I mentioned before I use the gesture navigation, so I have the navigation bar and the notification bar hidden (immersive mode),
So I tried hiding just the navigation bar and keeping the notification bar, now the keyboard shows up correctly without typing anything.

Therefore I think the problem is that the url toolbar is not able to detect the top of the keyboard correctly when the notification bar is hidden.
Maybe it messes with the keyboard height calculation, until I type something then suddenly it figures it out.

 its an annoying bug, but since I like having the notification bar hidden, so I think I will live with it :)
